NEWS.md

```diff
@@ -1,6 +1,36 @@
-## 5.8.0 - Unreleased
-* [MODSOURMAN-1022](https://issues.folio.org/browse/MODSOURMAN-1022) Remove step of initial saving of incoming records to SRS
+## 5.9.0 - Unreleased
+
+## 5.8.0 - Released (Quesnelia R1 2024)
```

Comment:
I think we need to mention somehow that DB is added to mod-invoice module, maybe it would be usefull for dev-ops

Reply:
added, thanks